src/run_releasenotes_check.py

```diff
@@ -0,0 +1,98 @@
+import logging
+import os
+import subprocess
+import sys
+from typing import Any, List
+
+from pytablewriter import MarkdownTableWriter
+
+from checkout_workflow.checkout_args import CheckoutArgs
+from git.git_repository import GitRepository
+from manifests.input_manifest import InputComponentFromSource, InputManifest
+from system import console
+from system.temporary_directory import TemporaryDirectory
+
+# in format yyyy-mm-dd, example 2022-07-26
+gitLogDate = os.getenv('GIT_LOG_DATE')
+# Github issue to add the generated markdown table as a new comment
+gitIssueNumber = os.getenv('GIT_ISSUE_NUMBER')
+# Boolen if set to null, just prints on the console.
+addComment = os.getenv('ADD_COMMENT', default="true")
+# Token used to add comment on an issue
+userToken = os.getenv('GITHUB_TOKEN')
+
+
+def main(gitLogDate: str, addComment: str) -> int:
+    if gitLogDate is None:
+        print("No GIT_LOG_DATE environmental value passed")
+        sys.exit()
+    value_matrix = []  # type: List[Any]
+    args = CheckoutArgs()
+    console.configure(level=args.logging_level)
+    manifest = InputManifest.from_file(args.manifest)
+    writer = MarkdownTableWriter(
+        table_name=f"{manifest.build.name} Commits(after {gitLogDate}) & Release Notes info",
+        headers=["Repo", "Branch", "CommitID", "Release Notes"],
+        value_matrix=value_matrix
+    )
+    with TemporaryDirectory(keep=False, chdir=True) as work_dir:
```

Comment:
Just remove it and let it use default (which is also false)

Reply:
@gaiksaya pushed the change in my latest commit:
https://github.com/opensearch-project/opensearch-build/issues/2345#issuecomment-1207891878